auth_token/contrib/is_core_auth/views.py

```diff
@@ -27,3 +71,13 @@ def get(self, request, *args, **kwargs):
         user = self.get_obj()
         takeover(request, user)
         return super().get(request, *args, **kwargs)
+
+
+class LoginCodeVerificationView(DjangoLoginView):
```

Comment:
We will have to add specific throttling for two factor in security

Reply:
I will solve it in a new issue.